components/automate-ui/src/app/pages/data-feed-create/data-feed-create.component.ts

```diff
@@ -166,6 +170,25 @@ export class DataFeedCreateComponent {
     }};
   }
 
+
+  showCustomField() {
+    this.showFields = {...this.showFields, ...{
+      name: true,
+      url: true,
+      authSelector: true,
+      tokenType: true,
+      token: true,
+      username: true,
+      password: true,
+      useHeaders: true
+    }};
+  }
+
+  updatePolicyCheckbox(event: boolean): void {
```

Comment:
can we rename this method to something else like updateHeaderCheckbox?

Reply:
Done